src/flatc.cpp

```diff
@@ -233,6 +244,9 @@ int main(int argc, const char *argv[]) {
       } else if(arg == "--version") {
         printf("flatc version %s\n", FLATC_VERSION);
         exit(0);
+      } else if(arg == "--grpc") {
+        grpc_enabled = true;
+        any_generator = true;
```

Comment:
probably `any_generator` should not be set here, since this relates to languages.